src/System/UnionMount.hs

```diff
@@ -57,8 +64,8 @@ mount ::
   FilePath ->
   -- | Only include these files (exclude everything else)
   [(b, FilePattern)] ->
-  -- | Ignore these patterns
-  [FilePattern] ->
+  -- | Ignore file pattern (optional)
+  Maybe FilePattern ->
```

Comment:
This should just be `Maybe FilePath`. The ignore file is relative to the root. It is not a "pattern" at all!